test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -639,6 +639,10 @@ public void PackCommand_PackProject_AddsProjectRefsAsPackageRefs()
         [PlatformTheory(Platform.Windows)]
         [InlineData("TargetFramework", "netstandard1.4")]
         [InlineData("TargetFrameworks", "netstandard1.4;net46")]
+#if NETCOREAPP5_0
```

Comment:
none of the pack tests actually test a net5.0 TFM with a platform identifier.

Reply:
We can't test this yet because the SDK doesn't recognize them yet, so I get:

```
error NETSDK1013: The TargetFramework value 'net5.0-windows10.0' was not recognized. It may be misspelled. If not, then the TargetFrameworkIdentifier and/or TargetFrameworkVersion properties must be specified explicitly.
```

...but they need our changes in order to add support on their end.

🙃

Reply:
yes, that is an excellent point :)